NotebookExamples/csharp/Samples/HousingML.ipynb

```diff
@@ -81,10 +81,9 @@
     }
    ],
    "source": [
-    "#r \"nuget:RestoreSources=https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json\"\n",
     "#r \"nuget:Microsoft.ML,1.4.0\"\n",
     "#r \"nuget:Microsoft.ML.AutoML,0.16.0\"\n",
-    "#r \"nuget:Microsoft.Data.Analysis,0.1.1-e191107-1\""
+    "#r \"nuget:Microsoft.Data.Analysis, 0.1.0\""
```

Comment:
Do we need/like the space between the comma and the version?

Reply:
Fixed